### PR TITLE
Update roles page

### PR DESCRIPTION
**Pull request by Benjaminpjacobs**
@alicetragedy this PR is in reference to Issue #405. All the bullet points in the issue have been addressed in addition to any grammar or spelling issues in those sections.

### CODE REVIEW

File: about/roles.md [hunk 1]
```diff
@@ -150,25 +145,25 @@ bacon-makers!
...
 
 Sponsors make it possible for Rails Girls Summer of Code to create a paid
-program for participants. Sponsors are sourced from all over the world, and
-represent an entire cross section of industry. While Sponsors receive a great
-deal of publicity and public exposure in return for their generous donations
-(usually a package) but most of the time not being involved actively sponsors
-take on a supporting actors role, but quite obviously a very importing one:
-They essentially pay for students so they are able to participate in this
-program.
+program for participants. Sourced from all of the world, sponsors represent
```
Comment (alicetragedy) [inline about/roles.md:148]: great change! 👍

File: about/roles.md [hunk 2]
```diff
@@ -150,25 +145,25 @@ bacon-makers!
...
 
 <h3 id="coaching_company">Coaching Companies</h3>
 
-This role is a formalization of a very successful concept from last year:
-Companies that fully host a students team for the duration of their project and
-allow employees to coach them. You can find out more about this initiative
-here: <a href="/guide/coaching-company">Coaching Companies</a>. Sponsors can be coaching
-companies too, as well as vice versa
+A Coaching Company is one that host an entire student team for the duration of 
+their project. In addition, they can allow employees to coach the participants.
```
Comment (alicetragedy) [inline about/roles.md]: could you remove “can” here? 
So basically: “In addition, they allow employees to coach the participants”

File: about/roles.md [hunk 3]
```diff
@@ -84,12 +84,16 @@ Should a coach not be available to help with an issue, students can also reach
...
 explained below.
 
-<h3 id="mentor">Mentor</h3>
+<h3 id="mentor">Mentor / Project Maintainer</h3>
 
 Mentors are experts on the Open Source project the team is working on. Ideally
 they are decision makers, or work closely with decision makers, so they can set
 general goals, give directions and provide feedback.
 
+At the beginning of summer Mentors will help students design a roadmap for the
```
Comment (alicetragedy) [inline about/roles.md]: could you remove “will” here? I think it's best if the whole page is in the present tense. 
Additionally, could you split this sentence into two? Something like:
“At the beginning of summer, mentors help students [...]. They are available throughout the summer to review pull requests, help with concepts, [...].”

File: about/roles.md [hunk 4]
```diff
@@ -84,12 +84,16 @@ Should a coach not be available to help with an issue, students can also reach
...
 
 Mentors are experts on the Open Source project the team is working on. Ideally
 they are decision makers, or work closely with decision makers, so they can set
 general goals, give directions and provide feedback.
 
+At the beginning of summer Mentors will help students design a roadmap for the 
+project and throughout the project be available to review pull requests, help
+with concepts and generally support the team.
+
 Since students will be supported by coaches in their day-to-day work, the
```
Comment (alicetragedy) [inline about/roles.md]: This wasn't in the original issue, but could you replace “will be” with “are”? I think most of the page is written in the present tense, so it's good to keep it consistent. Thanks!

File: about/roles.md [hunk 5]
```diff
@@ -150,25 +145,25 @@ bacon-makers!
...
 
 <h3 id="you">You</h3>
 
 Our <a href="/campaign">crowdfunding campaign</a> is vitally important to
 Summer of Code. Your donation goes directly to participants, and makes a big
-difference to how many can be accepted. If your company can't sponsor an entire
-package, they can still donate via our crowdfunding campaign.
+difference to how many can be accepted. If your company cannot sponsor an 
+entire package they can still donate a smaller amount via the campaign.
+Every and any donation counts!
```
Comment (alicetragedy) [inline about/roles.md:169]: 🎉

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @Benjaminpjacobs, thanks so much for your super quick contribution to the project! I've made a few comments on small things that need changing, but overall it's almost ready to go. Please make those changes when you have the time, and after that I think we'll be ready to go and merge this! ✨

**Review by alicetragedy — APPROVED** [review]
Looks like this is ready to go. thanks so much @Benjaminpjacobs 🎉

**Comment by Benjaminpjacobs** [comment]
Thank you @alicetragedy! Happy to help out. I'll keep an eye out if there's anything else I can help with.

**Comment by alicetragedy** [comment]
Sounds fantastic! I love having more contributors on board 👏

**Comment by alicetragedy** [comment]
closes #405 !

**Pull request merged**